app/views/api/v1/lookup_keys/show.json.rabl

```diff
@@ -1,3 +1,5 @@
-object @lookup_key
+object @lookup_key => :lookup_key
 
-attributes :key, :is_param, :required, :override, :description, :default_value, :id
\ No newline at end of file
+attributes :key, :required, :override, :description, :default_value, :id
```

Comment:
should we keep the is_param for compatibility? I'm pretty sure hammer might be using it?